app/src/main/java/nextstep/signup/ui/component/inputField/EmailInputField.kt

```diff
@@ -0,0 +1,66 @@
+package nextstep.signup.ui.component.inputField
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+fun EmailInputField(
+    value: String,
+    onValueChange: (String) -> Unit,
+    modifier: Modifier = Modifier,
+    hint: String = "",
+) {
+    TextField(
+        modifier = modifier
+            .then(Modifier.fillMaxWidth()),
```

Comment:
```suggestion
        modifier = modifier.fillMaxWidth(),
```
then을 사용하지 않고도 동일한 효과를 얻으실 수 있습니다.